.goreleaser.yml

```diff
@@ -0,0 +1,42 @@
+archives:
+  - name_template: '{{ .ProjectName }}_{{ .Version }}_{{ .Os }}_{{ .Arch }}'
+    format: zip
```

Comment:
GoReleaser packages `README` and `LICENSE` files by default here, which you may not want - at least I think we haven't packaged anything other than binaries so far for any providers.

```yaml
files: # Avoid packaging any extra (default) files
      - none*
```